Gemfile

```diff
@@ -1,8 +1,19 @@
 # frozen_string_literal: true
 
 source 'http://rubygems.org'
+
 gemspec
 
 # Use the most recent Rails release by default
 gem 'rails', '~> 7.0'
 gem 'sqlite3', '~> 1.4'
+
+gem 'rake', '~> 13.0'
+
+gem 'debug', '~> 1.8'
+gem 'rspec', '~> 3.12'
+gem 'rubocop', '~> 1.55'
+gem 'rubocop-performance', '~> 1.18'
+gem 'rubocop-rails', '~> 2.20'
+gem 'rubocop-rspec', '~> 2.22'
+gem 'simplecov', '>= 0.22.0'
```

Comment:
Are those necessary? Can they be pulled from the gemspec so that we want to update the gem versions, we don't need to do that in two places?

Reply:
done!